src/com/serotonin/mango/rt/dataImage/PointValueCache.java

```diff
@@ -35,104 +34,135 @@
 public class PointValueCache {
     private final int dataPointId;
     private final int defaultSize;
-    private final PointValueDao dao;
+    private int maxSize = 0;
+    private PointValueProxy pointValueProxy;
```

Comment:
Proxy is not a good name, I would suggest a service. Proxy is the name of the design pattern, and this class does not match this pattern